hazelcast-client/src/test/java/com/hazelcast/client/cache/nearcache/ClientCacheNearCacheBasicTest.java

```diff
@@ -128,6 +128,9 @@ public void tearDown() {
                 .setNearCacheManager(nearCacheManager)
                 .setCacheManager(cacheManager)
                 .setMemberCacheManager(memberCacheManager)
+                // FIXME: the JCache doesn't send invalidation on CREATED entries, so this will crash some tests
```

Comment:
AbstractRecordStore
```java
    protected R doPutRecord(Data key, R record, String source) {
        R oldRecord = records.put(key, record);
        if (oldRecord != null) {
            invalidateEntry(key, source);
        }
        return oldRecord;
    }
```
seems to prevent invalidation events on CREATED entries, so we cannot use the listener for now to wait for invalidation events after a JCache population